lang/cs/Org.Apache.REEF.Network/Group/Operators/Impl/ReduceSender.cs

```diff
@@ -40,6 +40,7 @@ public class ReduceSender<T> : IReduceSender<T>
         private const int PipelineVersion = 2;
         private readonly OperatorTopology<PipelineMessage<T>> _topology;
         private readonly PipelinedReduceFunction<T> _pipelinedReduceFunc;
+        private bool _isInitialize = false;
```

Comment:
rename all of these to `_isInitialized`. Also, this creates a potential threading hazard. Can we provide locking around it?